src/Controller/ProductController.php

```diff
@@ -35,10 +35,20 @@ class ProductController extends AbstractFOSRestController
      * @param Request $request
      * @param SerializerInterface $serializer
      * @return Response
+     * @throws InvalidArgumentException
      */
     public function viewProducts(ProductRepository $productRepository, Security $security, PaginatorInterface $pager,
                                  Request $request, SerializerInterface $serializer)
     {
+        $key = 'product.all';
```

Comment:
Ajouté le numéro de page dans ta clef pour gérer la pagination en cache ;(

Reply:
Il faut aussi penser à vider le cache lorsque tu ajoutes un produit ou que tu en supprimes un ;)

Reply:
> Ajouté le numéro de page dans ta clef pour gérer la pagination en cache ;(

Done

Reply:
> Il faut aussi penser à vider le cache lorsque tu ajoutes un produit ou que tu en supprimes un ;)

I have not implemented those 2 functions in my ProductController. Where should I create the function with $cache->delete($key) ?
Thank you.